packages/jazz-tools/src/tools/coValues/CoValueBase.ts

```diff
@@ -77,4 +77,15 @@ export abstract class CoValueJazzApi<V extends CoValue> {
 
     return new AnonymousJazzAgent(this.localNode);
   }
+
+  /**
+   * The timestamp of the last updated time of the CoValue
+   *
+   * Returns 0 if there are no updates.
```

Comment:
In the future (when all CoValues have a createdAt timestamp) I think it'd make more sense for lastUpdatedAt to match createdAt if there are no edits. But this makes sense for now

Reply:
In CoMap, where we expose `.$jazz.createdAt`, this is true.

But agree that making this now, might be a breaking change painful to deal with in the future.
Going to give another try at initializing the creation time tomorrow.

Reply:
Done!